spec/system/group_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "Group management", type: :system, js: true do
+  before do
+    Capybara.raise_server_errors = false
```

Comment:
Is using this fine? @junron
Instead, just move the ``time.js`` to ``public/js`` directory. That will solve the issue I guess.

Reply:
Great suggestion! I'll try it out